homeassistant/components/device_tracker/config_entry.py

```diff
@@ -165,9 +165,14 @@ def handle_device_event(ev: Event[EventDeviceRegistryUpdatedData]) -> None:
     hass.bus.async_listen(dr.EVENT_DEVICE_REGISTRY_UPDATED, handle_device_event)
 
 
+class BaseTrackerEntityDescription(EntityDescription, frozen_or_thawed=True):
```

Comment:
Don't we still want to allow changing the source type? There's a 3rd type that can be set for tracker entities. Or is this happening in another PR? There are many PRs now on this so I'm bit confused.

Reply:
It can still be set directly on the Entity via overriding the property or via the shorthand attribute.
I'm not convinced we need it on the Entity Description anymore.

Reply:
Ok.

Reply:
Then I think we don't need this base description. The base entity isn't supposed to be used directly (even though a couple of integrations do that).